$OpenBSD: patch-sample_server_c,v 1.1 2013/06/17 19:31:09 ajacoutot Exp $

XXX push upstream:
gssapi/gssapi_ext.h is MIT specific, so including it with Heimdal can cause compilation problems

--- sample/server.c.orig	Sat Jan 28 00:31:36 2012
+++ sample/server.c	Thu Jun 13 12:32:23 2013
@@ -85,7 +85,11 @@
 
 #ifdef HAVE_GSS_GET_NAME_ATTRIBUTE
 #include <gssapi/gssapi.h>
+# ifndef KRB5_HEIMDAL
+#  ifdef HAVE_GSSAPI_GSSAPI_EXT_H
 #include <gssapi/gssapi_ext.h>
+#  endif
+# endif
 #endif
 
 #include "common.h"
